tests/src/ch.qos.logback.contrib/logback-json-classic/0.1.5/src/test/java/ch_qos_logback_contrib/logback_json_classic/LogbackJsonClassicTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package ch_qos_logback_contrib.logback_json_classic;
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LogbackJsonClassicTest {
+
+    private static final String CONFIG_TAG = """
```

Comment:
Since all of the code in this class is same as in the [test above](https://github.com/oracle/graalvm-reachability-metadata/pull/378/files#diff-3bf893ed9ac4fff4a302b1e7805b0cce5ad0a26392840bcae2f8a1b32a56ada7R1) can they be merged into one test class that has just two different `CONFIG_TAG` variables to test these two cases